.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,57 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ main ]
```

Comment:
Does it mean the scan job is triggered for every PR? How long does it take for a scan?

Reply:
CodeQL scan job will be triggered for every PR and populates the detentions to "Security Alerts". It takes approximately 30mins for a scan. 
ACC security team will triage these alerts on a regular basis and triage the valid security issues by creating an GH issue.

Reply:
As discussed, can we remove the push and pr triggers and leave only the schedule trigger?